src/views/ForgetLogin.vue

```diff
@@ -39,27 +49,47 @@
   </div>
 </template>
 <script>
+import router from "@/router";
 
 export default {
   name: "ForgetLogin",
+  components: {},
+  data() {
+    return {
+      userId: "",
+      submitted: false,
+    };
+  },
+  methods: {
+    handleSubmit() {
+      this.submitted = true;
+      const { userId } = this;
+      const { dispatch } = this.$store;
+      if (userId) {
+        router.push({ name: "EnterOTP", params: { username: userId } });
+      } else {
+        alert("Enter you email or username");
+      }
+      dispatch("authStore/checkUser", userId);
+    },
+  },
 };
 </script>
 
 <style>
-.yo{
-    width: 400px;
-    margin-top: 100px;
-    margin-left: 50%;
-    margin-right: 50%;
+.yo {
```

Comment:
Please use verbose class names.

Reply:
@vikas339 you have been asked to use proper class names to maintain verbosity. Please do it.